db/migrate/20210927101340_create_bookings.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateBookings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bookings do |t|
+      t.references :user, foreign_key: true
+      t.references :meeting_room, foreign_key: true
+      t.datetime :start_time
+      t.datetime :end_time
+      t.date :date
```

Comment:
What is the purpose of having a date attribute here?

Reply:
Yes, actually It was optional attribute. Will remove it.